### PR TITLE
Fix result format to 1D

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Fix result format to 1D

libClient-py
https://github.com/acompany-develop/QuickMPC-libClient-py/pull/28
# Purpose
To reduce client-side memory
# Contents
Fix result format to 1D
Fix source code affected by format change
Fix test

# Testing Methods Performed
CI

### CODE REVIEW

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp [hunk 1]
```diff
@@ -109,13 +110,13 @@ TEST(ComputationToDbTest, SuccessReadShareLargeTest)
...
 TEST(ComputationToDbTest, SuccessReadModelParamTest)
 {
-    const std::string job_uuid = "ReadModelParamTestId";
+    const std::string job_uuid = "SuccessReadModelParamTest";
     initialize(job_uuid);
 
-    const std::string data = R"({"result":"[\"1\",\"2\",\"3\"]")"
-                             R"(,"meta":{"piece_id":0}})";
+    const std::string data = R"({"result":["1","2","3"])"
+                             R"(,"meta":{"column_number":4, "piece_id":0}})";
```
Comment (KotaTakahashi9320) [inline src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp:117]:
```suggestion
                             R"(,"meta":{"column_number":3, "piece_id":0}})";
```

File: src/ManageContainer/Server/LibcToManageContainer/L2m.go [hunk 2]
```diff
@@ -270,13 +274,12 @@ func (s *server) GetComputationResult(in *pb.GetComputationResultRequest, stream
 		logger_func("gRPC CheckProgress method with JobUUID: [%s] returns error: Code [%s](%d), Message [%s]", JobUUID, st.Code().String(), st.Code(), st.Message())
 	}
 
-	computationResults, computationErrInfo, err := s.m2dbclient.GetComputationResult(JobUUID)
+	ComputationResults, computationErrInfo, err := s.m2dbclient.GetComputationResult(JobUUID, []string{"dim1", "dim2", "schema"})
```
Comment (KotaTakahashi9320) [inline src/ManageContainer/Server/LibcToManageContainer/L2m.go:277]:
```suggestion
	computationResults, computationErrInfo, err := s.m2dbclient.GetComputationResult(JobUUID, []string{"dim1", "dim2", "schema"})
```

File: src/ManageContainer/Server/LibcToManageContainer/L2m.go [hunk 3]
```diff
@@ -289,29 +292,34 @@ func (s *server) GetComputationResult(in *pb.GetComputationResultRequest, stream
...
-	for _, result := range computationResults {
-		resultBytes, err := json.Marshal(result.Result)
-		if err != nil {
-			stream.Send(&pb.GetComputationResultResponse{
-				Message: "Internal Server Error",
-				IsOk:    false,
-				Result:  "",
-			})
-			return err
+	for _, result := range ComputationResults {
```
Comment (KotaTakahashi9320) [inline src/ManageContainer/Server/LibcToManageContainer/L2m.go:295]:
```suggestion
	for _, result := range computationResults {
```

File: src/ManageContainer/Server/LibcToManageContainer/L2m.go [hunk 4]
```diff
@@ -289,29 +292,34 @@ func (s *server) GetComputationResult(in *pb.GetComputationResultRequest, stream
...
+		response := pb.GetComputationResultResponse{
+			Message:      "ok",
+			IsOk:         true,
+			Status:       ComputationResults[0].Status,
```
Comment (KotaTakahashi9320) [inline src/ManageContainer/Server/LibcToManageContainer/L2m.go:299]:
```suggestion
			Status:       computationResults[0].Status,
```

**Review by arukuka — APPROVED** [review]
Thank you for your contribution!

**Review by mdonaka — APPROVED** [review]

**Pull request merged**